exchange/auction.go

```diff
@@ -147,6 +147,17 @@ func (a *auction) doCache(ctx context.Context, cache prebid_cache_client.Client,
 		}
 	}
 
+	if len(debugLog) > 0 {
+		if jsonBytes, err := json.Marshal(debugLog); err == nil {
+			toCache = append(toCache, prebid_cache_client.Cacheable{
+				Type:       prebid_cache_client.TypeXML,
```

Comment:
Consider changing debugLog to a struct which includes the same string value you have right now and the type. That'll make it reusable for, say, the AMP endpoint if it needs to do something similar in the future for json logs. Same for the TTL.

Reply:
Added a `debugLog` type that holds all info, including if the debug is enabled

Reply:
Sorry missed adding this change in. Will be added with unit test update.

Reply:
Change added